src/tags/tags.toml

```
@@ -33,6 +33,7 @@ _ _
 - **[Radon ᴱ ᴬ](<https://github.com/EvolutionX-10/Radon>)**
 - **[Sapphire Application Commands Examples ᴱ](<https://github.com/vladfrangu/sapphire-application-commands-examples>)**
 - **[Archangel ᴱ ᴰ](<https://github.com/favware/archangel>)**
+- **[Zeyr ᴱ ᴰ ᴬ](<https://github.com/brxem/zeyr>)**
```

```suggestion
- **[Zeyr ᴰ ᴬ](<https://github.com/brxem/zeyr>)**
```

Your bot used CommonJS, not ESM. That is to say, when compiled the JavaScript is CJS. If it would be ESM you'd have to setup things like `"type": "module"`, import mapping or imports with file extensions, etc.